modules/fieldlinessequence/rendering/renderablefieldlinessequence.h

```diff
@@ -130,91 +145,88 @@ class RenderableFieldlinesSequence : public Renderable {
+    properties::Vec4Property _colorUniform;
     // Whether or not to use additive blending
-    properties::BoolProperty _pColorABlendEnabled;
+    properties::BoolProperty _colorABlendEnabled;
 
 // Whether or not to use Domain
-    properties::BoolProperty _pDomainEnabled;
+    properties::BoolProperty _domainEnabled;
     // Group to hold the Domain properties
-    properties::PropertyOwner _pDomainGroup;
+    properties::PropertyOwner _domainGroup;
     // Domain Limits along x-axis
-    properties::Vec2Property _pDomainX;
+    properties::Vec2Property _domainX;
     // Domain Limits along y-axis
-    properties::Vec2Property _pDomainY;
+    properties::Vec2Property _domainY;
     // Domain Limits along z-axis
-    properties::Vec2Property _pDomainZ;
+    properties::Vec2Property _domainZ;
     // Domain Limits radially
-    properties::Vec2Property _pDomainR;
+    properties::Vec2Property _domainR;
 
 // Simulated particles' color
-    properties::Vec4Property _pFlowColor;
+    properties::Vec4Property _flowColor;
     // Toggle flow [ON/OFF]
-    properties::BoolProperty _pFlowEnabled;
+    properties::BoolProperty _flowEnabled;
     // Group to hold the flow/particle properties
-    properties::PropertyOwner _pFlowGroup;
+    properties::PropertyOwner _flowGroup;
     // Size of simulated flow particles
-    properties::IntProperty _pFlowParticleSize;
+    properties::IntProperty _flowParticleSize;
     // Size of simulated flow particles
-    properties::IntProperty _pFlowParticleSpacing;
+    properties::IntProperty _flowParticleSpacing;
     // Toggle flow direction [FORWARDS/BACKWARDS]
-    properties::BoolProperty _pFlowReversed;
+    properties::BoolProperty _flowReversed;
     // Speed of simulated flow
-    properties::IntProperty _pFlowSpeed;
+    properties::IntProperty _flowSpeed;
 
     // Whether or not to use masking
-    properties::BoolProperty _pMaskingEnabled;
+    properties::BoolProperty _maskingEnabled;
     // Group to hold the masking properties
-    properties::PropertyOwner _pMaskingGroup;
-    // Lower limit for allowed values
-    properties::StringProperty _pMaskingMin;
-    // Upper limit for allowed values
-    properties::StringProperty _pMaskingMax;
+    properties::PropertyOwner _maskingGroup;
+    // Lower and upper range limit for allowed values
+    properties::Vec2Property _maskingMinMax;
     // Index of the extra quantity to use for masking
-    properties::OptionProperty _pMaskingQuantity;
+    properties::OptionProperty _maskingQuantity;
+    // used to save property for later initialization
+    int _maskingQuantityTemp = 0;
 
-// Button which sets camera focus to parent node of the renderable
-    properties::TriggerProperty _pFocusOnOriginBtn;
+    /// Line width for the line rendering part
+    properties::FloatProperty _lineWidth;
     // Button which executes a time jump to start of sequence
-    properties::TriggerProperty _pJumpToStartBtn;
+    properties::TriggerProperty _jumpToStartBtn;
```

Comment:
I think we probably want to get rid of this and replace it with an action for the specific usecase instead?    Not sure about that though

Reply:
I think all time depended renderables should have a button like this by default. Leaving it as it is.